code/__HELPERS/mobs.dm

```diff
@@ -162,9 +162,8 @@ proc/age2agedescription(age)
 			break
 
 		if(!QDELETED(Tloc) && (QDELETED(target) || Tloc != target.loc))
-			if(Uloc != Tloc || Tloc != user)
```

Comment:
Is this necessary for this bugfix? If not, could you atomize it?

Reply:
fine